### PR TITLE
[MM-59060] Remove pre release features section from advanced section of user settings

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
We removed this section from user settings as they were not used at all since the beginning of them being added. We used markdown preview for it at some point but that was only used in the channel header and no where else.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-59060

#### Screenshots
none

#### Release Note

```release-note
Removed pre-release features section due to non usage.
```

### CODE REVIEW

**Review by larkox — APPROVED** [review]
Great cleanup!

File: webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx [hunk 1]
```diff
@@ -701,79 +616,6 @@ export default class AdvancedSettingsDisplay extends React.PureComponent<Props,
...
-            const inputs = [];
-
-            this.state.preReleaseFeaturesKeys.forEach((key) => {
-                const feature = this.state.preReleaseFeatures[key as keyof PreReleaseFeaturesType];
-                inputs.push(
-                    <div key={'advancedPreviewFeatures_' + feature.label}>
-                        <div className='checkbox'>
-                            <label>
-                                <input
-                                    id={'advancedPreviewFeatures' + feature.label}
```
Comment (hmhealey) [inline webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx:718]: It looks like there's a commented out reference to these settings in `e2e-tests/cypress/tests/integration/channels/accessibility/accessibility_account_settings_spec.js`. Would you mind removing that?

**Review by hmhealey — APPROVED** [review]
Thanks for cleaning this up! I found one more spot where these seem to be used, but this otherwise looks good to me

**Comment by hmhealey** [comment]
Actually, I found a few other spots that reference these prerelease features. Most of them reference either the `FeatureTogglePrefix` constant or `feature_toggle_markdown_preview`.

1. `isFeatureEnabled` in `utils/utils` isn't used any more and neither is the client-side version of the `FeatureTogglePrefix` constant
2. There's a `apiSaveShowMarkdownPreviewPreference` utility in the Cypress tests which enables the previews in a few E2E tests
3. There's references to the `feature_enabled_markdown_preview` preference in the user import/export code on the server. There's also a server-side version of `model.FeatureTogglePrefix` along with that
4. Related to that, there's a whole `SqlPreferenceStore.deleteUnusedFeatures` method for cleaning up old unused preferences in that category which doesn't seem like it'd do anything any more.

The server-side ones could possibly be left to XYZ since I'd assume they'd want to be aware of any changes to the bulk import code like that

**Comment by M-ZubairAhmed** [comment]
> Actually, I found a few other spots that reference these prerelease features. Most of them reference either the `FeatureTogglePrefix` constant or `feature_toggle_markdown_preview`.
> 
>     1. `isFeatureEnabled` in `utils/utils` isn't used any more and neither is the client-side version of the `FeatureTogglePrefix` constant
> 
>     2. There's a `apiSaveShowMarkdownPreviewPreference` utility in the Cypress tests which enables the previews in a few E2E tests
> 
>     3. There's references to the `feature_enabled_markdown_preview` preference in the user import/export code on the server. There's also a server-side version of `model.FeatureTogglePrefix` along with that
> 
>     4. Related to that, there's a whole `SqlPreferenceStore.deleteUnusedFeatures` method for cleaning up old unused preferences in that category which doesn't seem like it'd do anything any more.
> 
> 
> The server-side ones could possibly be left to XYZ since I'd assume they'd want to be aware of any changes to the bulk import code like that

Nice find i have removed all related code except for server import code

**Pull request merged**